modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -950,6 +994,8 @@ public void stop() {
 
         busyLock.block();
 
+        metaStorageMgr.unregisterWatch(zonesWatchListener);
```

Comment:
Could you please also add the unregister for rebalance watch?

Reply:
I removed unregisterWatch for zonesWatchListener because as I understand it is not necessary.

Reply:
It's a secret knowledge that meta storage manager aggregates all watches into big one and unregister it on ms.stop. We shouldn't rely on such side-effetcts. All in all that may change in future,

Reply:
Ok. I've added "unregister" for all watches.